404.html

```diff
@@ -1,27 +1,39 @@
 ---
-layout: page
-title: "Page non trouvée"
+layout: default
 permalink: /404.html
 hide: true
 ---
-<div class="container">
-	<div style="text-align:center;margin-bottom:4em;">Toutes nos excuses, mais la page que vous avez demandé n'a pu être trouvée.
-Vous pouvez utiliser la recherche pour lire un article en particulier ou vous laisser guider par la liste des derniers articles.</div>
-
-
-    <section class="slice">
-	    <div class="read-also container">
-	        <h3 class="read-also-title">Derniers articles :</h3>
-	        <ul class="posts read-also-list">
-			    {% for post in site.posts limit:5 %}
-			    	<li class="read-also-item">
-			            <a class="read-also-link" href="{{ post.url | prepend: site.baseurl_root }}">
-			                {{ post.title }}
-			            </a>
-			        </li>
-			    {% endfor %}
-	        </ul>
-	    </div>
-    </section>
+<div class="container not-found">
+	<h1 class="md-content__title">Page not found</h1>
+	<div class="md-content">
+		We apologize for any inconvenience, but the page you requested was not found. 
```

Comment:
"you've requested"

Reply:
you requested is correct nope ?

Reply:
Effectivement, ici c'est la même chose ! 
https://ell.stackexchange.com/questions/6635/attached-is-a-copy-youve-requested